<!DOCTYPE html>
<html>
  <head>
    <!-- 开始：HEAD 部分 -->
    <meta charset="utf-8">
    <title>Group - RNA-Puzzles</title>
    <meta name="author" content="Chichau Miao">

    <!-- 移动设备优化相关 Meta 标签 -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- SEO 相关：Canonical URL、Favicon、RSS 订阅链接 -->
    <link rel="canonical" href="http://www.rnapuzzles.org/group/">
    <link href="/favicon.png" type="image/png" rel="icon">
    <link href="/atom.xml" rel="alternate" title="RNA-Puzzles" type="application/atom+xml">

    <!-- Open Graph / Twitter Meta 标签 -->
    <meta name="twitter:card" content="summary_large_image">
    <meta property="og:type" content="website">
    <meta property="og:url" content="http://www.rnapuzzles.org/group/">
    <meta property="og:title" content="Group - RNA-Puzzles">
    
    <!-- 引入 jQuery -->
    <script src="/js/jquery-3.2.1.min.js"></script>

    <!-- 引入 CSS 文件 -->
    <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="/css/stylish-portfolio.css" rel="stylesheet" type="text/css">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
    <link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">
    <link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">
    <!-- 结束：HEAD 部分 -->
</head>
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Das Group</title>
  <meta name="description" content="The lab website of Zhichao Miao lab.
">
  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="/_posts/groups/2017-11-16-das-group.html">
  <link rel="shortcut icon" type="image/x-icon" href="/favicon.png">
  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
  

  <!-- 将汉堡按钮的 CSS 样式放在 style 标签内 -->
  <style>
    .navbar-toggle .icon-bar {
      background-color: #eeeeee;
    }
  </style>

</head>


  <body>

    <!-- 导航条容器：使用 Bootstrap 的 navbar 类，设置导航条扩展模式、深色主题以及内联样式 -->
<div class="navbar navbar-expand-lg navbar-dark" style="border-radius: 0; border: 0; background-color: #337ab7" role="navigation">
	<!-- 限制内容宽度为 1200px 的 fluid 容器 -->
	<div class="container-fluid" style="max-width: 1500px;">
		<!-- 导航头部：包含汉堡按钮和品牌链接 -->
		<div class="navbar-header">
			<!-- 响应式切换按钮（汉堡菜单）
			     当屏幕较小时，该按钮显示，点击后控制右侧所有链接、RSS 和搜索框的折叠/展开
			     为使按钮图标颜色与当前字体颜色（白色）一致，在 CSS 中设置 .navbar-toggle .icon-bar 的背景色 -->
			<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
				<span class="sr-only">Toggle navigation</span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
			</button>

			<!-- 品牌链接：使用 Flexbox 布局使品牌文字和 logo 分布在两侧 -->
			<a class="navbar-brand" href="/" style="display: flex; justify-content: space-between; align-items: center;">
				<b style="font-size: 32px; color: #eeeeee;">RNA-Puzzles</b>
				<img src="/images/header/logo.jpg" style="height: 53px; margin-top: 0; margin-bottom: 0; padding: 0px; border-radius: 0;" alt="logo">
			</a>
		</div> <!-- end of .navbar-header -->

		<!-- 可折叠的导航菜单部分：包括多个页面链接、搜索框和 RSS 订阅按钮 -->
		<div class="collapse navbar-collapse" id="navbar-collapse-1">
			<!-- 右对齐的导航列表 -->
			<ul class="nav navbar-nav navbar-right">
				<!-- 页面链接 -->
				<li>
					<a class="nav-link-1" href="/results">Results</a>
				</li>
				<li>
					<a class="nav-link-1" href="/open-puzzle">Open Puzzles</a>
				</li>
				<li>
					<a class="nav-link-1" href="/group">Groups</a>
				</li>
				<li>
					<a class="nav-link-1" href="/publication">Publications</a>
				</li>
				<li>
					<a class="nav-link-1" href="https://chichaumiau.github.io/rnapuzzlestoolkit/">Resources</a>
				</li>
				<!-- 搜索框：
				     使用 form 表单，置于一个 <li> 内与其他链接同一行显示，
				     通过 position: relative; top: 2px; 实现向下偏移 2px，
				     同时保证输入框垂直居中 -->
				<li>
					<form class="navbar-form" role="search" action="/search" method="GET" style="margin: 0; padding: 0;">
						<div class="form-group">
							<input type="text" class="form-control" name="q" placeholder="search" style="color: #999; vertical-align: middle; position: relative; top: 9px;">
						</div>
					</form>
				</li>
				<!-- RSS 订阅按钮：
				     为避免上下空白，通过清除图片的默认 margin/padding，并设置 display: inline-block，
				     同时保持 vertical-align: middle 使其垂直居中 -->
				<li>
					<a class="subscribe-rss" href="/atom.xml" title="subscribe via RSS" style="line-height: 1;">
						<span class="visible-xs">RSS</span>
						<img class="hidden-xs" src="/images/header/rss.png" alt="RSS" style="vertical-align: middle; margin: 0; padding: 0; display: inline-block;">
					</a>
				</li>
			</ul>
		</div> <!-- end of .collapse .navbar-collapse -->
	</div> <!-- end of .container-fluid -->
</div> <!-- end of .navbar -->


  <!-- 使用 container 保证内容区域居中 -->
  <div class="container">
    <div class="row">
      <!-- 主页内容区域：占用 8 栏 -->
      <div id="homeid" class="col-md-8">
        <article class="post" itemscope itemtype="http://schema.org/BlogPosting">

  <header class="post-header">
    <h1 class="post-title" itemprop="name headline">Das Group</h1>
    <p class="post-meta"><time datetime="2017-11-18T19:08:41+08:00" itemprop="datePublished">Nov 18, 2017</time></p>
  </header>

  <div class="post-content" itemprop="articleBody">
    <!--
[<i class="fa fa-envelope-square fa-fw fa-2x"></i>](mailto:wj_hust08@hust.edu.cn?cc=yxiao@hust.edu.cn)
-->
<p><a href="https://profiles.stanford.edu/rhiju-das" target="_balnk"><strong>Das</strong> </a> group is from Howard Hughes Medical Institute and Department of Biochemistry,Stanford University.
<a href="https://daslab.stanford.edu/" target="_blank"><strong>&lt;/br&gt;(Click here to find more information about Das  Group)</strong></a>
&lt;/br&gt;</p>

<h2>Prediction Approaches:</h2>
<p>The Das lab submitted models for the majority of the Puzzles in Round V. The exceptions were two puzzles in which Das lab members were involved in experimental structure determination of the target or a near-identical molecule (PZ30, human telomerase; and PZ31, the SARS-CoV-2 frameshift stimulation element) and two puzzles that were also targets in the CASP15 competition (puzzles 35 and 36, CPEB ribozymes), for which Das lab members served as assessors. For the remaining 17 of 21 Puzzles, the Das lab made use of tools developed in the Rosetta3 codebase.</p>

<p>Fragment assembly of RNA with full atom refinement (FARFAR) has been the lab’s primary RNA modeling method since the beginning of the RNA Puzzles, with some automation piloted in the RNAWorks3D server and several further improvements collected in the FARFAR2 method, used in RNA PZ22 and later. For most of the Round V puzzles, more than one set of models was submitted, with one set selected manually (typically by R.D. in collaboration with lab members, tagged as Das), another selected based on the Rosetta energy (typically tagged as FARFAR2), and other sets selected by scoring from a tensor field network developed by the Dror group, tagged as “TFN”, or by the eventual name of the network, “ARES”, for Atomic Rotationally Equivariant Scorer). FARFAR(2) was used to submit at least one set of models for 16 of the 17 RNA Puzzles for which any models were submitted by the Das lab. In addition to the FARFAR-based models, for 8 of the 17 RNA Puzzles, the Das lab submitted models from a different Rosetta-based approach, Stepwise Monte Carlo (SWM). This method carries out high resolution conformational search of loops or junctions without use of fragments of prior RNA structures, guided by the all-atom Rosetta energy function, but its application is restricted to targets where only one such unknown loop or junction needs to be built. For Puzzle 11, our lab acquired chemical mapping data and released these data with anonymized sequences to the community in the following entries in the RNA Mapping DataBase: RNAPZ11_STD_0001, DMS, CMCT, and the SHAPE modifier 1M7; RNAPZ11_1M7_0001, RNAPZ11_1M7_0002, RNAPZ11_1M7_0003, for mutate-and-map data with three sets of mutations per nucleotide. De-anonymized versions of these datasets are now available at RNAPZ11_STD_0002, RNAPZ11_1M7_0004, RNAPZ11_1M7_0005, RNAPZ11_1M7_0006.</p>

<p>For all RNA puzzles to date, including targets for which the most accurate overall model was submitted, the Das lab has been able to identify background literature describing the function and, typically, candidate secondary structures or other constraints useful for modeling. For PZ18 (Zika xrRNA), a domain-swapped crystal structure of a different xrRNA served as a template, but expert input was required to inform which regions could be used as templates and which needed to be built de novo with SWM, which was being developed concomitantly with PZ18. For PZ22 (Hatchet ribozyme), secondary structures from RNAfold were refined based on manual inspection of covariance in published sequence alignments. For PZ23 (iMango-III), potential G-quadruplexes as well as the binding site of the ligand were hypothesized after inspection of previously solved Mango aptamers, and these hypotheses were used to seed distinct SWM runs that led to each of the submissions.  For PZ24 (VA-I), a pseudoknotted secondary structure had been proposed in the literature and was used to seed FARFAR2 modeling, followed by ARES scoring, which was being developed concomitantly through PZ24-28 in close collaboration with the Dror lab. For Puzzles 26-28 (T-box/tRNA complexes), secondary structures, including the set of intermolecular base pairs between the tRNA CAA tail and the T-box, and templates from previously solved fragments of tRNA/T-box complexes were identified manually and used in FARFAR2 modeling, followed by ARES scoring. For PZ34 (methyltransferase ribozyme), special SWM runs were carried out that modeled the RNA with the enzymatic product m1A at the methylation site, and manual model selection took into consideration the steric accessibility for an exogenous O6-methylguanine as the methyl donor.</p>

<h2>Publications:</h2>
<ol>
  <li>Das R, Kretsch RC, Simpkin AJ, Mulvaney T, Pham P, Rangan R, Bu F, Keegan RM, Topf M, Rigden DJ, Miao Z, Westhof E. <a href="https://pubmed.ncbi.nlm.nih.gov/37876231/">Assessment of three-dimensional RNA structure prediction in CASP15. Proteins.</a> 2023 Dec;91(12):1747-1770.</li>
  <li>Leaver-Fay A, Tyka M, Lewis SM, Lange OF, Thompson J, Jacak R, Kaufman K, Renfrew PD, Smith CA, Sheffler W, Davis IW, Cooper S, Treuille A, Mandell DJ, Richter F, Ban YE, Fleishman SJ, Corn JE, Kim DE, Lyskov S, Berrondo M, Mentzer S, Popović Z, Havranek JJ, Karanicolas J, Das R, Meiler J, Kortemme T, Gray JJ, Kuhlman B, Baker D, Bradley P. <a href="https://pubmed.ncbi.nlm.nih.gov/21187238/">ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules.</a> Methods Enzymol. 2011;487:545-74.</li>
  <li>Das R, Karanicolas J, Baker D. <a href="https://pubmed.ncbi.nlm.nih.gov/20190761/">Atomic accuracy in predicting and designing noncanonical RNA structure.</a> Nat Methods. 2010 Apr;7(4):291-4.</li>
  <li>Miao Z, Adamiak RW, Antczak M, Boniecki MJ, Bujnicki J, Chen SJ, Cheng CY, Cheng Y, Chou FC, Das R, Dokholyan NV, Ding F, Geniesse C, Jiang Y, Joshi A, Krokhotin A, Magnus M, Mailhot O, Major F, Mann TH, Piątkowski P, Pluta R, Popenda M, Sarzynska J, Sun L, Szachniuk M, Tian S, Wang J, Wang J, Watkins AM, Wiedemann J, Xiao Y, Xu X, Yesselman JD, Zhang D, Zhang Y, Zhang Z, Zhao C, Zhao P, Zhou Y, Zok T, Żyła A, Ren A, Batey RT, Golden BL, Huang L, Lilley DM, Liu Y, Patel DJ, Westhof E. <a href="https://pubmed.ncbi.nlm.nih.gov/32371455/">RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers.</a> RNA. 2020 Aug;26(8):982-995.</li>
  <li>Watkins AM, Rangan R, Das R. <a href="https://pubmed.ncbi.nlm.nih.gov/32531203/">FARFAR2: Improved De Novo Rosetta Prediction of Complex Global RNA Folds. Structure.</a> 2020 Aug 4;28(8):963-976.e6.</li>
  <li>Townshend RJL, Eismann S, Watkins AM, Rangan R, Karelina M, Das R, Dror RO. <a href="https://pubmed.ncbi.nlm.nih.gov/34446608/">Geometric deep learning of RNA structure.</a> Science. 2021 Aug 27;373(6558):1047-1051. doi: 10.1126/science.abe5650. Erratum in: Science. 2023 Jan 27;379(6630):eadg6616.</li>
  <li>Watkins AM, Geniesse C, Kladwang W, Zakrevsky P, Jaeger L, Das R. <a href="https://pubmed.ncbi.nlm.nih.gov/29806027/">Blind prediction of noncanonical RNA structure at atomic accuracy.</a> Sci Adv. 2018 May 25;4(5):eaar5316.</li>
  <li>Cordero P, Kladwang W, VanLang CC, Das R. <a href="https://pubmed.ncbi.nlm.nih.gov/22913637/">Quantitative dimethyl sulfate mapping for automated RNA secondary structure inference.</a> Biochemistry. 2012 Sep 11;51(36):7037-9.</li>
  <li>Cordero P, Kladwang W, VanLang CC, Das R. <a href="(https://pubmed.ncbi.nlm.nih.gov/24136598/)">The mutate-and-map protocol for inferring base pairs in structured RNA.</a> Methods Mol Biol. 2014;1086:53-77.</li>
  <li>Tian S, Das R. <a href="https://pubmed.ncbi.nlm.nih.gov/28453672/">Primerize-2D: automated primer design for RNA multidimensional chemical mapping.</a>  Bioinformatics. 2017 May 1;33(9):1405-1406.</li>
  <li>Gruber AR, Lorenz R, Bernhart SH, Neuböck R, Hofacker IL. <a href="https://pubmed.ncbi.nlm.nih.gov/18424795/">The Vienna RNA websuite.</a> Nucleic Acids Res. 2008 Jul 1;36(Web Server issue):W70-4.</li>
</ol>

<!--
32.	Das, R. et al. Assessment of three-dimensional RNA structure prediction in CASP15. Proteins 91, 1747–1770 (2023).
33.	Leaver-Fay, A. et al. ROSETTA3: an object-oriented software suite for the simulation and design of macromolecules. Methods Enzymol. 487, 545–574 (2011).
34.	Das, R., Karanicolas, J. & Baker, D. Atomic accuracy in predicting and designing noncanonical RNA structure. Nat. Methods 7, 291–294 (2010).
35.	Miao, Z. et al. RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers. RNA 26, 982–995 (2020).
36.	Watkins, A. M., Rangan, R. & Das, R. FARFAR2: Improved De Novo Rosetta Prediction of Complex Global RNA Folds. Structure 28, 963–976.e6 (2020).
37.	Townshend, R. J. L. et al. Geometric deep learning of RNA structure. Science 373, 1047–1051 (2021).
38.	Watkins, A. M. et al. Blind prediction of noncanonical RNA structure at atomic accuracy. Sci Adv 4, eaar5316 (2018).
39.	Cordero, P., Kladwang, W., VanLang, C. C. & Das, R. Quantitative dimethyl sulfate mapping for automated RNA secondary structure inference. Biochemistry 51, 7037–7039 (2012).
40.	Cordero, P., Kladwang, W., VanLang, C. C. & Das, R. The mutate-and-map protocol for inferring base pairs in structured RNA. Methods Mol. Biol. 1086, 53–77 (2014).
41.	Tian, S. & Das, R. Primerize-2D: automated primer design for RNA multidimensional chemical mapping. Bioinformatics 33, 1405–1406 (2017).
42.	Gruber, A. R., Lorenz, R., Bernhart, S. H., Neuböck, R. & Hofacker, I. L. The Vienna RNA Websuite. Nucleic Acids Res. 36, W70–W74 (2008).
-->

  </div>

</article>

      </div>
      <!-- 侧边栏区域：占用 4 栏 -->
      <div id="newsid" class="col-md-4">
        <!-- Begin: Sidebar 模块 -->
<aside class="sidebar col-sm-44" style="padding: 15px;">
	    <!-- CSS Files -->
		<link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
		<link href="/css/stylish-portfolio.css" rel="stylesheet" type="text/css">
		<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
		<link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">
		<link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">
	<!-- Begin: 关于信息部分 -->
	<section>
		<h1>RNA-Puzzles</h1>
		<p>A collective blind evaluation of RNA three-dimensional structure prediction</p>
		<!-- GitHub 链接 -->
		<a href="https://github.com/RNA-Puzzles">
			<i class="fa fa-github-square fa-fw fa-2x"></i>
		</a>
		<!-- Email 链接 -->
		<a href="mailto:ibmc.cnrs@gmail.com">
			<i class="fa fa-envelope-square fa-fw fa-2x"></i>
		</a>
		<!-- Twitter 链接 -->
		<a href="https://twitter.com/RNAPuzzles">
			<i class="fa fa-twitter-square fa-fw fa-2x"></i>
		</a>
		<!-- Coffee 链接（示例链接，可根据需要修改） -->
		<a href="">
			<i class="fa fa-coffee fa-fw fa-2x"></i>
		</a>



	</section>
	<!-- End: 关于信息部分 -->

	<!-- Begin: 最近文章模块 -->
	<section class="panel panel-info">
		<div class="panel-heading">
			<h3 class="panel-title">Recent Posts</h3>
		</div>
		<div id="recent_posts" class="list-group">
			<a class="list-group-item" href="/news/2023/10/07/Submission-Process.html">Submission Process</a>
			<a class="list-group-item" href="/news/2023/10/07/Research-Spectrum.html">Research Spectrum</a>
			<a class="list-group-item" href="/news/2023/10/07/Recent-Engagements.html">Recent Engagements</a>
			<a class="list-group-item" href="/news/2023/10/07/Participating-Teams.html">Participating Teams</a>
			<a class="list-group-item" href="/news/2023/10/07/Methodological-Progress.html">Methodological Progress</a>
		</div>
	</section>
	<!-- End: 最近文章模块 -->

	<!-- Begin: GitHub Repos 模块 -->
	<section class="panel panel-info clearfix">
		<div class="panel-heading">
			<h3 class="panel-title">GitHub Repos</h3>
		</div>
		<div class="list-group" id="gh_repos">
			<p class="loading">Status updating...</p>
		</div>
		<div class="gh-profile-link pull-right text-muted">
			<a href="https://github.com/RNA-Puzzles">@RNA-Puzzles</a> on GitHub
		</div>
		<!-- 使用 jQuery 动态加载 GitHub 仓库信息 -->
		<script type="text/javascript">
			$(document).ready(function(){
				// 如果未加载 jXHR 库，则动态加载
				if (!window.jXHR){
					var jxhr = document.createElement('script');
					jxhr.type = 'text/javascript';
					jxhr.src = '/javascripts/libs/jXHR.js';
					var s = document.getElementsByTagName('script')[0];
					s.parentNode.insertBefore(jxhr, s);
				}
				// 调用 github.showRepos() 函数显示仓库列表
				github.showRepos({
					user: 'RNA-Puzzles',
					count: 0,
					skip_forks: true,
					target: '#gh_repos'
				});
			});
		</script>
		<script src="/javascripts/github.js" type="text/javascript"></script>
	</section>
	<!-- End: GitHub Repos 模块 -->

	<!-- Begin: 分类模块 -->
	<section class="panel panel-info clearfix">
		<div class="panel-heading">
			<h3 class="panel-title">Categories</h3>
		</div>
		<div class="list-group-item">
			<a href="/blog/group_category/">group (16)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/news_category/">news (7)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/results_category/">results (39)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/tables_category/">table (129)</a>
		</div>
	</section>
	<!-- End: 分类模块 -->
</aside>
<!-- End: Sidebar 模块 -->

      </div>
    </div>
  </div>

    <div id="footer" class="panel">
  <div class="panel-footer" style='background-color: #333333; color: #eeeeee;'>
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">
			
		  <p>&copy 2025 RNA-Puzzles. </p>
		  <p>We are part of the <a href="https://gzlab.ac.cn">Guangzhou Laboratory</a> and <a href="http://www.rnapuzzles.org/">RNApuzzles</a>.</p>
		  <p>Site made with <a href="https://jekyllrb.com">Jekyll</a>;  <a href="/aboutwebsite.html">copy and  modify it for your own research group.</a></p>
		  <p>  </p>
		  <p>  </p>
		</div>
		
		<div class="col-sm-4">
		  Funding:<br />
          - <a href="https://gzlab.ac.cn">R&D Programs of Guangzhou Laboratory</a><br />
          - <a href="http://www.most.gov.cn/">MOST IT+BT projects</a> <br />
          - <a href="https://www.nsfc.gov.cn/">NSFC</a> <br />	  
		</div>
		
		<div class="col-sm-4">
		  Contact:<br />
		  Building B2, Guangzhou National Laboratory<br />
		  96 Xingdao South Road, Guangzhou International Bio Island, <br />
		  Haizhu District, Guangzhou, Guangdong, China.<br />
          (<a href="https://goo.gl/maps/hDLr6siEAS2Eaypt8">Maps</a>)
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="/js/bootstrap.min.js"></script>


  </body>

</html>
