<!DOCTYPE html>
<html>
  <head>
    <!-- 开始：HEAD 部分 -->
    <meta charset="utf-8">
    <title>Group - RNA-Puzzles</title>
    <meta name="author" content="Chichau Miao">

    <!-- 移动设备优化相关 Meta 标签 -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <!-- SEO 相关：Canonical URL、Favicon、RSS 订阅链接 -->
    <link rel="canonical" href="http://www.rnapuzzles.org/group/">
    <link href="/favicon.png" type="image/png" rel="icon">
    <link href="/atom.xml" rel="alternate" title="RNA-Puzzles" type="application/atom+xml">

    <!-- Open Graph / Twitter Meta 标签 -->
    <meta name="twitter:card" content="summary_large_image">
    <meta property="og:type" content="website">
    <meta property="og:url" content="http://www.rnapuzzles.org/group/">
    <meta property="og:title" content="Group - RNA-Puzzles">
    
    <!-- 引入 jQuery -->
    <script src="/js/jquery-3.2.1.min.js"></script>

    <!-- 引入 CSS 文件 -->
    <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
    <link href="/css/stylish-portfolio.css" rel="stylesheet" type="text/css">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
    <link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">
    <link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">
    <!-- 结束：HEAD 部分 -->
</head>
  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Miao Lab - Publications</title>
  <meta name="description" content="Miao Lab -- Publications.">
  <link rel="stylesheet" href="/css/main.css">
  <link rel="canonical" href="/publication/">
  <link rel="shortcut icon" type="image/x-icon" href="/favicon.png">
  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
  

  <!-- 将汉堡按钮的 CSS 样式放在 style 标签内 -->
  <style>
    .navbar-toggle .icon-bar {
      background-color: #eeeeee;
    }
  </style>

</head>


  <body>

    <!-- 导航条容器：使用 Bootstrap 的 navbar 类，设置导航条扩展模式、深色主题以及内联样式 -->
<div class="navbar navbar-expand-lg navbar-dark" style="border-radius: 0; border: 0; background-color: #337ab7" role="navigation">
	<!-- 限制内容宽度为 1200px 的 fluid 容器 -->
	<div class="container-fluid" style="max-width: 1500px;">
		<!-- 导航头部：包含汉堡按钮和品牌链接 -->
		<div class="navbar-header">
			<!-- 响应式切换按钮（汉堡菜单）
			     当屏幕较小时，该按钮显示，点击后控制右侧所有链接、RSS 和搜索框的折叠/展开
			     为使按钮图标颜色与当前字体颜色（白色）一致，在 CSS 中设置 .navbar-toggle .icon-bar 的背景色 -->
			<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar-collapse-1" aria-expanded="false">
				<span class="sr-only">Toggle navigation</span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
			</button>

			<!-- 品牌链接：使用 Flexbox 布局使品牌文字和 logo 分布在两侧 -->
			<a class="navbar-brand" href="/" style="display: flex; justify-content: space-between; align-items: center;">
				<b style="font-size: 32px; color: #eeeeee;">RNA-Puzzles</b>
				<img src="/images/header/logo.jpg" style="height: 53px; margin-top: 0; margin-bottom: 0; padding: 0px; border-radius: 0;" alt="logo">
			</a>
		</div> <!-- end of .navbar-header -->

		<!-- 可折叠的导航菜单部分：包括多个页面链接、搜索框和 RSS 订阅按钮 -->
		<div class="collapse navbar-collapse" id="navbar-collapse-1">
			<!-- 右对齐的导航列表 -->
			<ul class="nav navbar-nav navbar-right">
				<!-- 页面链接 -->
				<li>
					<a class="nav-link-1" href="/results">Results</a>
				</li>
				<li>
					<a class="nav-link-1" href="/open-puzzle">Open Puzzles</a>
				</li>
				<li>
					<a class="nav-link-1" href="/group">Groups</a>
				</li>
				<li>
					<a class="nav-link-1" href="/publication">Publications</a>
				</li>
				<li>
					<a class="nav-link-1" href="https://chichaumiau.github.io/rnapuzzlestoolkit/">Resources</a>
				</li>
				<!-- 搜索框：
				     使用 form 表单，置于一个 <li> 内与其他链接同一行显示，
				     通过 position: relative; top: 2px; 实现向下偏移 2px，
				     同时保证输入框垂直居中 -->
				<li>
					<form class="navbar-form" role="search" action="/search" method="GET" style="margin: 0; padding: 0;">
						<div class="form-group">
							<input type="text" class="form-control" name="q" placeholder="search" style="color: #999; vertical-align: middle; position: relative; top: 9px;">
						</div>
					</form>
				</li>
				<!-- RSS 订阅按钮：
				     为避免上下空白，通过清除图片的默认 margin/padding，并设置 display: inline-block，
				     同时保持 vertical-align: middle 使其垂直居中 -->
				<li>
					<a class="subscribe-rss" href="/atom.xml" title="subscribe via RSS" style="line-height: 1;">
						<span class="visible-xs">RSS</span>
						<img class="hidden-xs" src="/images/header/rss.png" alt="RSS" style="vertical-align: middle; margin: 0; padding: 0; display: inline-block;">
					</a>
				</li>
			</ul>
		</div> <!-- end of .collapse .navbar-collapse -->
	</div> <!-- end of .container-fluid -->
</div> <!-- end of .navbar -->


  <!-- 使用 container 保证内容区域居中 -->
  <div class="container">
    <div class="row">
      <!-- 主页内容区域：占用 8 栏 -->
      <div id="homeid" class="col-md-8">
        <div id="gridid" class="col-sm-12">
  <head>
    <!-- Begin: HEAD Section -->
    <meta charset="utf-8" />
    <title>RNA-Puzzles</title>
    <meta name="author" content="Chichau Miao" />

    <!-- Mobile Optimization Meta Tags -->
    <meta name="HandheldFriendly" content="True" />
    <meta name="MobileOptimized" content="320" />
    <meta name="viewport" content="width=device-width, initial-scale=1" />

    <!-- SEO: Canonical URL, Favicon, and Alternate Feed -->
    <link rel="canonical" href="http://www.rnapuzzles.org/" />
    <link href="/favicon.png" type="image/png" rel="icon" />
    <link href="/atom.xml" rel="alternate" title="RNA-Puzzles" type="application/atom+xml" />

    <!-- Open Graph / Twitter Meta Tags -->
    <meta name="twitter:card" content="summary_large_image" />
    <meta property="og:type" content="website" />
    <meta property="og:url" content="http://www.rnapuzzles.org/" />
    <meta property="og:title" content="RNA-Puzzles" />
    <meta property="og:description" content="RNA-Puzzles a community-wide blind evaluation of RNA 3D structure prediction." />

    <!-- jQuery -->
    <script src="/js/jquery-3.2.1.min.js"></script>

    <!-- CSS Files -->
    <link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css" />
    <link href="/css/stylish-portfolio.css" rel="stylesheet" type="text/css" />
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
    <link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet" />
    <link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css" />
    <!-- End: HEAD Section -->
</head>

<h1>Publications</h1>

<p>(# co-first author, * corresponding author)</p>

<h2>Group highlights</h2>

<p>(For a full list of publications and patents see <a href="#full-list-of-publications">below</a> or go to 
<a href="https://scholar.google.com/citations?user=OUFQCssAAAAJ">Google Scholar</a>, 
<a href="https://orcid.org/0000-0002-5777-9815">ORCID</a>)</p>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>RNA-Puzzles Round V: blind predictions of 23 RNA structures</pubtit>
      <p><img src="/images/pubpic/RNA-Puzzles_V.png" class="img-responsive" width="33%" style="float: left" /></p>
      <p></p>
      <p><b>Bu F</b>, Adam Y, Adamiak RW, Antczak M, de Aquino BRH, Badepally NG, Batey RT, Baulin EF, Boinski P, Boniecki MJ, Bujnicki JM, Carpenter KA, Chacon J, Chen SJ, Chiu W, Cordero P, Das NK, Das R, Dawson WK, DiMaio F, Ding F, Dock-Bregeon AC, Dokholyan NV, Dror RO, Dunin-Horkawicz S, Eismann S, Ennifar E, Esmaeeli R, Farsani MA, Ferré-D’Amaré AR, Geniesse C, Ghanim GE, Guzman HV, Hood IV, Huang L, Jain DS, Jaryani F, Jin L, Joshi A, Karelina M, Kieft JS, Kladwang W, Kmiecik S, Koirala D, Kollmann M, Kretsch RC, Kurciński M, Li J, Li S, Magnus M, Masquida B, Moafinejad SN, Mondal A, Mukherjee S, Nguyen THD, Nikolaev G, Nithin C, Nye G, Pandaranadar Jeyeram IPN, Perez A, Pham P, Piccirilli JA, Pilla SP, Pluta R, Poblete S, Ponce-Salvatierra A, Popenda M, Popenda L, Pucci F, Rangan R, Ray A, Ren A, Sarzynska J, Sha CM, Stefaniak F, Su Z, Suddala KC, Szachniuk M, Townshend R, Trachman RJ 3rd, Wang J, Wang W, Watkins A, Wirecki TK, Xiao Y, Xiong P, Xiong Y, Yang J, Yesselman JD, Zhang J, Zhang Y, Zhang Z, Zhou Y, Zok T, Zhang D, Zhang S, Żyła A, Westhof E*, <b>Miao Z*</b></p>
      <p><em><strong><a href="https://www.nature.com/articles/s41592-024-02543-9">Nat Methods. (2024). doi:10.1038/s41592-024-02543-9.</a></strong></em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers.</pubtit>
      <p><img src="/images/pubpic/RNA-Puzzles_IV.jpg" class="img-responsive" width="33%" style="float: left" /></p>
      <p></p>
      <p><b>Miao, Z.</b>, Adamiak, R. W., Antczak, M., Boniecki, M. J., Bujnicki, J., Chen, S.-J., Cheng, C. Y., Cheng, Y., Chou, F.-C., Das, R., Dokholyan, N. V., Ding, F., Geniesse, C., Jiang, Y., Joshi, A., Krokhotin, A., Magnus, M., Mailhot, O., Major, F., Mann, T. H., Piątkowski, P., Pluta, R., Popenda, M., Sarzynska, J., Sun, L., Szachniuk, M., Tian, S., Wang, J., Wang, J., Watkins, A. M., Wiedemann, J., Xiao, Y., Xu, X., Yesselman, J. D., Zhang, D., Zhang, Y., Zhang, Z., Zhao, C., Zhao, P., Zhou, Y., Zok, T., Żyła, A., Ren, A., Batey, R. T., Golden, B. L., Huang, L., Lilley, D. M., Liu, Y., Patel, D. J. &amp; Westhof, E.#</p>
      <p><em><strong><a href="https://pubmed.ncbi.nlm.nih.gov/32371455/">RNA 26, 982–995 (2020).</a></strong></em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme.</pubtit>
      <p><img src="/images/pubpic/RNA-Puzzles_III.jpg" class="img-responsive" width="33%" style="float: left" /></p>
      <p></p>
      <p><b>Miao, Z.</b>, Adamiak, R. W., Antczak, M., Batey, R. T., Becka, A. J., Biesiada, M., Boniecki, M. J., Bujnicki, J. M., Chen, S.-J., Cheng, C. Y., Chou, F.-C., Ferré-D’Amaré, A. R., Das, R., Dawson, W. K., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Geniesse, C., Kappel, K., Kladwang, W., Krokhotin, A., Łach, G. E., Major, F., Mann, T. H., Magnus, M., Pachulska-Wieczorek, K., Patel, D. J., Piccirilli, J. A., Popenda, M., Purzycka, K. J., Ren, A., Rice, G. M., Santalucia, J., Jr, Sarzynska, J., Szachniuk, M., Tandon, A., Trausch, J. J., Tian, S., Wang, J., Weeks, K. M., Williams, B., 2nd, Xiao, Y., Xu, X., Zhang, D., Zok, T. &amp; Westhof, E.#</p>
      <p><em><strong><a href="https://pubmed.ncbi.nlm.nih.gov/28138060/">RNA 23, 655–672 (2017).</a></strong></em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> highlighted by <a href="https://facultyopinions.com/article/727254907">Faculty Opinions</a></p>
    </div>
  </div>

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures.</pubtit>
      <p><img src="/images/pubpic/RNA-Puzzles_II.jpg" class="img-responsive" width="33%" style="float: left" /></p>
      <p></p>
      <p><b>Miao, Z.</b>, Adamiak, R. W., Blanchet, M.-F., Boniecki, M., Bujnicki, J. M., Chen, S.-J., Cheng, C., Chojnowski, G., Chou, F.-C., Cordero, P., Cruz, J. A., Ferré-D’Amaré, A. R., Das, R., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Kladwang, W., Krokhotin, A., Lach, G., Magnus, M., Major, F., Mann, T. H., Masquida, B., Matelska, D., Meyer, M., Peselis, A., Popenda, M., Purzycka, K. J., Serganov, A., Stasiewicz, J., Szachniuk, M., Tandon, A., Tian, S., Wang, J., Xiao, Y., Xu, X., Zhang, J., Zhao, P., Zok, T. &amp; Westhof, E.#</p>
      <p><em><strong><a href="https://pubmed.ncbi.nlm.nih.gov/25883046/">RNA 21, 1066–1084 (2015).</a></strong></em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<div class="row">

  <div class="col-sm-6 clearfix">
    <div class="well">
      <pubtit>RNA-Puzzles: a CASP-like evaluation of RNA three-dimensional structure prediction.</pubtit>
      <p><img src="/images/pubpic/RNA-Puzzles_I.jpg" class="img-responsive" width="33%" style="float: left" /></p>
      <p></p>
      <p>Cruz JA, Blanchet MF, Boniecki M, Bujnicki JM, Chen SJ, Cao S, Das R, Ding F, Dokholyan NV, Flores SC, Huang L, Lavender CA, Lisi V, Major F, Mikolajczak K, Patel DJ, Philips A, Puton T, Santalucia J, Sijenyi F, Hermann T, Rother K, Rother M, Serganov A, Skorupski M, Soltysinski T, Sripakdeevong P, Tuszynska I, Weeks KM, Waldsich C, Wildauer M, Leontis NB, Westhof E.#</p>
      <p><em><strong><a href="https://rnajournal.cshlp.org/content/18/4/610.long/">RNA. 2012 Apr;18(4):610-25.</a></strong></em></p>
      <p class="text-danger"><strong> </strong></p>
      <p> </p>
    </div>
  </div>

</div>

<p>   </p>

<h2>Full List of publications</h2>

<p><b>RNA-Puzzles Round V: blind predictions of 23 RNA structures</b> <br />
  <b>Bu F</b>, Adam Y, Adamiak RW, Antczak M, de Aquino BRH, Badepally NG, Batey RT, Baulin EF, Boinski P, Boniecki MJ, Bujnicki JM, Carpenter KA, Chacon J, Chen SJ, Chiu W, Cordero P, Das NK, Das R, Dawson WK, DiMaio F, Ding F, Dock-Bregeon AC, Dokholyan NV, Dror RO, Dunin-Horkawicz S, Eismann S, Ennifar E, Esmaeeli R, Farsani MA, Ferré-D’Amaré AR, Geniesse C, Ghanim GE, Guzman HV, Hood IV, Huang L, Jain DS, Jaryani F, Jin L, Joshi A, Karelina M, Kieft JS, Kladwang W, Kmiecik S, Koirala D, Kollmann M, Kretsch RC, Kurciński M, Li J, Li S, Magnus M, Masquida B, Moafinejad SN, Mondal A, Mukherjee S, Nguyen THD, Nikolaev G, Nithin C, Nye G, Pandaranadar Jeyeram IPN, Perez A, Pham P, Piccirilli JA, Pilla SP, Pluta R, Poblete S, Ponce-Salvatierra A, Popenda M, Popenda L, Pucci F, Rangan R, Ray A, Ren A, Sarzynska J, Sha CM, Stefaniak F, Su Z, Suddala KC, Szachniuk M, Townshend R, Trachman RJ 3rd, Wang J, Wang W, Watkins A, Wirecki TK, Xiao Y, Xiong P, Xiong Y, Yang J, Yesselman JD, Zhang J, Zhang Y, Zhang Z, Zhou Y, Zok T, Zhang D, Zhang S, Żyła A, Westhof E*, <b>Miao Z*</b> <br /><em><a href="https://www.nature.com/articles/s41592-024-02543-9">Nat Methods. (2024). doi:10.1038/s41592-024-02543-9.</a></em></p>

<p><b>Exome sequencing identifies homozygous variants in MBOAT7 associated with neurodevelopmental disorder</b> <br />
  Nazmina, G., Khan, A.#, Jiang, J., Miao, Z., Khan, SN., Khan, MI., Shah, AH., Shah, AH., Khisroon, M.#, Haack, TB. <br /><em><a href="https://onlinelibrary.wiley.com/doi/10.1111/cge.14469">Clin Genet. 10.1111/cge.14469(2023).</a></em></p>

<p><b>Regulatory circular RNAs in viral diseases: applications in diagnosis and therapy.</b> <br />
  Wang, W., Sun, L., Huang, M., Quan, Y., Jiang, T., <b>Miao, Z.#</b>, Zhang, Q.# <br /><em><a href="https://www.tandfonline.com/doi/full/10.1080/15476286.2023.2272118">RNA Biol. doi.org/10.1080/15476286.2023.2272118 (2023).</a></em></p>

<p><b>Assessment of three-dimensional RNA structure prediction in CASP15.</b> <br />
  Das, R.#, Kretsch, RC., Simpkin, AJ., Mulvaney, T., Pham, P., Rangan, R., <b>Bu, F.</b>, Keegan, RM., Topf, M., Rigden, DJ.,<b>Miao, Z.#</b>, Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/37876231/">Proteins. 10.1002/prot.26602(2023).</a></em></p>

<p><b>Nr4a1 marks a distinctive ILC2 activation subset in the mouse inflammatory lung.</b> <br />
  Xu, S., Zhang, Y., Liu, X., Liu, H., Zou, X., Zhang, L., Wang, J., Zhang, Z., Xu, X., Li, M., Li, K., Shi, S., Zhang, Y., <b>Miao, Z.</b>, Zha, J., Yu, Y. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/37833706/">BMC Biol. 10.1186/s12915-023-01690-3(2023).</a></em></p>

<p><b>Benchmarking strategies for cross-species integration of single-cell RNA sequencing data.</b> <br />
  Song, Y., <b>Miao, Z.</b>, Brazma, A., Papatheodorou, I. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/37838716/">Nat Commun. 10.1038/s41467-023-41855-w(2023)</a></em></p>

<p><b>Ribocentre-switch: a database of riboswitches.</b> <br />
  Bu, F., Lin, X., Liao, W., Lu, Z., He, Y., Luo, Y., Peng, X., Li, M., Huang, Y., Chen, X., Xiao, B., Jiang, J., Deng, J., Huang, J., Lin, T., <b>Miao, Z.#</b>, Huang, L.# <br /><em><a href="https://doi.org/10.1093/nar/gkad891">Nucleic Acids Res. gkad891 (2023).</a></em></p>

<p><b>Single Cell Analysis of the Fate of Injected Oncogenic RasV12 Cells in Adult Wild Type Drosophila.</b> <br />
  Chen, D., Lan, X., Huang, X., Huang, J., Zhou, X., <b>Miao, Z.</b>, Ma, Y., Goto, A., Ji, S., &amp; Hoffmann, J. A. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/36996781/">J Innate Immun, 15(1), 442–467(2023).</a></em></p>

<p><b>Structural basis of sRNA RsmZ regulation of Pseudomonas aeruginosa virulence.</b> <br />
  Jia, X., Pan, Z., Yuan, Y., Luo, B., Luo, Y., Mukherjee, S., Jia, G., Liu, L., Ling, X., Yang, X., <b>Miao, Z.</b>, Wei, X., Bujnicki, J.M., Zhao, K., Su, Z. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/36828938/">Cell Res. 10.1038/s41422-023-00786–3 (2023).</a></em></p>

<p><b>Sterile 20-like kinase 3 promotes tick-borne encephalitis virus assembly by interacting with NS2A and prM and enhancing the NS2A-NS4A association.</b> <br />
  Tang, J., Xu, C., Fu, M., Liu, C., Zhang, X., Zhang, W., Pei, R., Wang, Y., Zhou, Y., Chen, J., <b>Miao, Z.</b>, Pan, G., Yang, Q., Chen, X. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/36840407/">J Med Virol. 10.1002/jmv.28610 (2023).</a></em></p>

<p><b>RNA 3D Structure Comparison Using RNA-Puzzles Toolkit</b> <br />
  Magnus, M., &amp; <b>Miao, Z.#</b> <br /><em><a href="https://link.springer.com/protocol/10.1007/978-1-0716-2768-6_16#citeas">RNA Structure Prediction. Springer US, 263-285 (2023).</a></em></p>

<p><b>Defects and asymmetries in the visual pathway of non-human primates with natural strabismus and amblyopia</b> <br />
  Liu, F., Wang, Z., Huang, W., Xu, Y., Sang, X., Liu, R., Li, Z., Bi, Y., Tang, L., Peng, J., Wei, J., <b>Miao, Z.</b>, Yan, J., &amp; Liu, S. <br /><em><a href="https://doi.org/10.24272/j.issn.2095-8137.2022.254">Zoological Research. 44(1):153-168 (2023).</a></em></p>

<p><b>Identification of visual cortex cell types and species differences using single-cell RNA sequencing</b> <br />
  Wei, J., Hao, Z., Xu, C., Huang, M., Tang, L., Xu, N., Liu, R., Shen, Y., Teichmann, S. A., <b>Miao, Z.#</b>, &amp; Liu, S. <br /><em><a href="https://www.nature.com/articles/s41467-022-34590-1">Nature Communications. 13(1):1-21 (2022).</a></em></p>

<p><b>Ribocentre: a database of ribozymes.</b> <br />
  Deng, J., Shi, Y., Peng, X., He, Y., Chen, X., Li, M., Lin, X., Liao, W., Huang, Y., Jiang, T., Lilley, D. M. J., <b>Miao, Z.#</b>, &amp; Huang, L.# <br /><em><a href="https://doi.org/10.1093/nar/gkac840">Nucleic Acids Res. gkac840 (2022).</a></em></p>

<p><b>Single-cell atlas of human liver development reveals pathways directing hepatic cell fates.</b> <br />
  Wesley, B. T., Ross, A. D. B., Muraro, D., <b>Miao, Z.</b>, Saxton, S., Tomaz, R. A., Morell, C. M., Ridley, K., Zacharis, E. D., Petrus-Reurer, S., Kraiczy, J., Mahbubani, K. T., Brown, S., Garcia-Bernardo, J., Alsinet, C., Gaffney, D., Horsfall, D., Tysoe, O. C., Botting, R. A., Stephenson, E., Popescu, D.-M., MacParland, S., Bader, G., McGilvray, I. D., Ortmann, D., Sampaziotis, F., Saeb-Parsy, K., Haniffa, M., Stevens, K. R., Zilbauer, M., Teichmann, S. A., &amp; Vallier, L.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/36109670/">Nat Cell Biol. 1476-4679 (2022).</a></em></p>

<p><b>Linking transcriptomes with morphological and functional phenotypes in mammalian retinal ganglion cells.</b> <br />
  Huang, W., Xu, Q., Su, J., Tang, L., Hao, Z.-Z., Xu, C., Liu, R., Shen, Y., Sang, X., Xu, N., Tie, X., <b>Miao, Z.^</b>, Liu, X., Xu, Y., Liu, F., Liu, Y., &amp; Liu, S.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/36103830/">Cell Rep. 40(11):111322 (2022).</a></em></p>

<p><b>Computational Pipeline for Reference-Free Comparative Analysis of RNA 3D Structures Applied to SARS-CoV-2 UTR Models. International Journal of Molecular Sciences.</b> <br />
  Gumna J., Antczak M., Adamiak RW., Bujnicki JM., Chen S-J., Ding F., Ghosh P., Li J., Mukherjee S., Nithin C., Pachulska-Wieczorek K., Ponce-Salvatierra A., Popenda M., Sarzynska J., Wirecki T., Zhang D., Zhang S., Zok T., Westhof E.,  <b>Miao Z.#</b>, Szachniuk M., &amp; Rybarczyk A.# <br /><em><a href="https://doi.org/10.3390/ijms23179630">Models. Int. J. Mol. Sci.23, 9630 (2022).</a></em></p>

<p><b>Single-cell transcriptomics of adult macaque hippocampus reveals neural precursor cell populations.</b> <br />
  Hao, Z.-Z., Wei, J.-R., Xiao, D., Liu, R., Xu, N., Tang, L., Huang, M., Shen, Y., Xing, C., Huang, W., Liu, X., Xiang, M.#, Liu, Y.#, <b>Miao, Z.#</b> &amp; Liu, S.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/35637371/">Nat. Neurosci. 1–13 (2022).</a></em></p>

<p><b>Identification of TPBG-Expressing Amacrine Cells in DAT-tdTomato Mouse.</b> <br />
  Huang, W., Xu, Q., Liu, F., Su, J., Xiao, D., Tang, L., Hao, Z.-Z., Liu, R., Xiang, K., <b>Bi, Y., Miao, Z.</b>, Liu, X., Liu, Y. &amp; Liu, S.# <br /><em><a href="https://iovs.arvojournals.org/article.aspx?articleid=2778820">Invest. Ophthalmol. Vis. Sci. 63, 13–13 (2022).</a></em></p>

<p><b>Single-cell genomics identifies distinct B1 cell developmental pathways and reveals aging-related changes in the B-cell receptor repertoire.</b> <br />
  Luo, Y., Wang, J., Li, K., Li, M., Xu, S., Liu, X., Zhang, Z., Xu, X., Zhang, Y., Pan, J., Liu, P., Gao, S., <b>Miao, Z.#</b> &amp; Yu, Y.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/35526067/">Cell Biosci. 12, 57 (2022).</a></em></p>

<p><b>Expression Atlas update: gene and protein expression in multiple species.</b> <br />
  Moreno, P., Fexova, S., George, N., Manning, J. R., <b>Miao, Z.</b>, Mohammed, S., Muñoz-Pomer, A., Fullgrabe, A., Bi, Y., Bush, N., Iqbal, H., Kumbham, U., Solovyev, A., Zhao, L., Prakash, A., García-Seisdedos, D., Kundu, D. J., Wang, S., Walzer, M., Clarke, L., Osumi-Sutherland, D., Tello-Ruiz, M. K., Kumari, S., Ware, D., Eliasova, J., Arends, M. J., Nawijn, M. C., Meyer, K., Burdett, T., Marioni, J., Teichmann, S., Vizcaíno, J. A., Brazma, A.# &amp; Papatheodorou, I.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/34850121/">Nucleic Acids Res. 50, D129–D140 (2022)</a></em></p>

<p><b>Single-Cell Analysis Reveals the Immune Characteristics of Myeloid Cells and Memory T Cells in Recovered COVID-19 Patients With Different Severities.</b> <br />
  Li, X.^, <b>Garg, M.^</b>, Jia, T., Liao, Q., Yuan, L., Li, M., Wu, Z., Wu, W., Bi, Y., George, N., Papatheodorou, I., Brazma, A., Luo, H., Fang, S., <b>Miao, Z.#</b> ,Shu, Y.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/35046942/">Frontiers in Immunology 12, 1664-3224 (2022).</a></em></p>

<p><b>Evaluation of the stereochemical quality of predicted RNA 3D models in the RNA-Puzzles submissions.</b> <br />
  Carrascoza, F., Antczak, M., <b>Miao, Z.#</b>, Westhof, E. &amp; Szachniuk, M.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/34819324/">RNA (2021). doi:10.1261/rna.078685.121.</a></em></p>

<p><b>Meta-analysis of COVID-19 single-cell studies confirms eight key immune responses.</b> <br />
  <b>Garg, M.^</b>, Li, X.^, Moreno, P., Papatheodorou, I., Shu, Y.#, Brazma, A.# &amp; <b>Miao, Z.#</b> <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/34675242/">Sci. Rep. 11, 20833 (2021).</a></em></p>

<p><b>Mapping Rora expression in resting and activated CD4+ T cells.</b> <br />
  Haim-Vilmovsky, L., Henriksson, J., Walker, J. A., <b>Miao, Z.#</b>, Natan, E., Kar, G., Clare, S., Barlow, J. L., Charidemou, E., Mamanova, L., Chen, X., Proserpio, V., Pramanik, J., Woodhouse, S., Protasio, A. V., Efremova, M., Griffin, J. L., Berriman, M., Dougan, G., Fisher, J., Marioni, J. C., McKenzie, A. N. J. &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/34003838/">PLoS One 16, e0251233 (2021).</a></em></p>

<p><b>High-throughput full-length single-cell RNA-seq automation.</b> <br />
  Mamanova, L.#^, <b>Miao, Z.^</b>, Jinat, A., Ellis, P., Shirley, L. &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/33990801/">Nat. Protoc. 16, 2886–2915 (2021).</a></em></p>

<p><b>User-friendly, scalable tools and workflows for single-cell RNA-seq analysis.</b> <br />
  Moreno, P., Huang, N., Manning, J. R., Mohammed, S., Solovyev, A., Polanski, K., Bacon, W., Chazarra, R., Talavera-López, C., Doyle, M. A., Marnier, G., Grüning, B., Rasche, H., George, N., Fexova, S. K., Alibi, M., <b>Miao, Z.</b>, Perez-Riverol, Y., Haeussler, M., Brazma, A., Teichmann, S., Meyer, K. B. &amp; Papatheodorou, I.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/33782609/">Nat. Methods 18, 327–328 (2021).</a></em></p>

<p><b>Longevity, clonal relationship, and transcriptional program of celiac disease-specific plasma cells.</b> <br />
  Lindeman, I., Zhou, C., Eggesbø, L. M., <b>Miao, Z.</b>, Polak, J., Lundin, K. E. A., Jahnsen, J., Qiao, S.-W., Iversen, R. &amp; Sollid, L. M.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/33095260/">J. Exp. Med. 218(2):e20200852, (2021).</a></em></p>

<p><b>Single-cell meta-analysis of SARS-CoV-2 entry genes across tissues and demographics.</b> <br />
  Muus, C., Luecken, M. D., Eraslan, G., Sikkema, L., Waghray, A., Heimberg, G., Kobayashi, Y., Vaishnav, E. D., Subramanian, A., Smillie, C., Jagadeesh, K. A., Duong, E. T., Fiskin, E., Triglia, E. T., Ansari, M., Cai, P., Lin, B., Buchanan, J., Chen, S., Shu, J., Haber, A. L., Chung, H., Montoro, D. T., Adams, T., Aliee, H., Allon, S. J., Andrusivova, Z., Angelidis, I., Ashenberg, O., Bassler, K., Bécavin, C., Benhar, I., Bergenstråhle, J., Bergenstråhle, L., Bolt, L., Braun, E., Bui, L. T., Callori, S., Chaffin, M., Chichelnitskiy, E., Chiou, J., Conlon, T. M., Cuoco, M. S., Cuomo, A. S. E., Deprez, M., Duclos, G., Fine, D., Fischer, D. S., Ghazanfar, S., Gillich, A., Giotti, B., Gould, J., Guo, M., Gutierrez, A. J., Habermann, A. C., Harvey, T., He, P., Hou, X., Hu, L., Hu, Y., Jaiswal, A., Ji, L., Jiang, P., Kapellos, T. S., Kuo, C. S., Larsson, L., Leney-Greene, M. A., Lim, K., Litviňuková, M., Ludwig, L. S., Lukassen, S., Luo, W., Maatz, H., Madissoon, E., Mamanova, L., Manakongtreecheep, K., Leroy, S., Mayr, C. H., Mbano, I. M., McAdams, A. M., Nabhan, A. N., Nyquist, S. K., Penland, L., Poirion, O. B., Poli, S., Qi, C., Queen, R., Reichart, D., Rosas, I., Schupp, J. C., Shea, C. V., Shi, X., Sinha, R., Sit, R. V., Slowikowski, K., Slyper, M., Smith, N. P., Sountoulidis, A., Strunz, M., Sullivan, T. B., Sun, D., Talavera-López, C., Tan, P., Tantivit, J., Travaglini, K. J., Tucker, N. R., Vernon, K. A., Wadsworth, M. H., Waldman, J., Wang, X., Xu, K., Yan, W., Zhao, W., Ziegler, C. G. K., NHLBI LungMap Consortium &amp; Human Cell Atlas Lung Biological Network. <b>Miao, Z.</b> as a consortium author in Human Cell Atlas Lung Biological Network. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/33654293/">Nat. Med. 27, 546–559 (2021).</a></em></p>

<p><b>NCKAP1L defects lead to a novel syndrome combining immunodeficiency, lymphoproliferation, and hyperinflammation.</b> <br />
  Castro, C. N., Rosenzwajg, M., Carapito, R., Shahrooei, M., Konantz, M., Khan, A., <b>Miao, Z.</b>, Groß, M., Tranchant, T., Radosavljevic, M., Paul, N., Stemmelen, T., Pitoiset, F., Hirschler, A., Nespola, B., Molitor, A., Rolli, V., Pichot, A., Faletti, L. E., Rinaldi, B., Friant, S., Mednikov, M., Karauzum, H., Aman, M. J., Carapito, C., Lengerke, C., Ziaee, V., Eyaid, W., Ehl, S., Alroqi, F., Parvaneh, N. &amp; Bahram, S. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32766723/">J. Exp. Med. 217(12):e20192275, (2020).</a></em></p>

<p><b>SARS-CoV-2 entry factors are highly expressed in nasal epithelial cells together with innate immune genes.</b> <br />
  Sungnak, W., Huang, N., Bécavin, C., Berg, M., Queen, R., Litvinukova, M., Talavera-López, C., Maatz, H., Reichart, D., Sampaziotis, F., Worlock, K. B., Yoshida, M., Barnes, J. L. &amp; Human Cell Atlas Lung Biological Network. <b>Miao, Z.</b> as a consortium author in Human Cell Atlas Lung Biological Network. <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32327758/">Nat. Med. 26(5):681-687 (2020).</a></em></p>

<p><b>Two Cases of Recessive Intellectual Disability Caused by NDST1 and METTL23 Variants.</b> <br />
  Khan, A., <b>Miao, Z.</b>, Umair, M., Ullah, A., Alshabeeb, M. A., Bilal, M., Ahmad, F., Rappold, G. A., Ansar, M. &amp; Carapito, R.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32878022/">Genes (Basel) 11(9):1021 (2020).</a></em></p>

<p><b>Secondary structure of the SARS-CoV-2 5’-UTR.</b> <br />
  <b>Miao, Z.</b>, Tidu, A., Eriani, G. &amp; Martin, F.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32965173/">RNA Biol. 18, 447–456 (2021).</a></em></p>

<p><b>Advances in RNA 3D Structure Modeling Using Experimental Data.</b> <br />
  Li, B., Cao, Y., Westhof, E. &amp; <b>Miao, Z.#</b> <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/33193680/">Front. Genet. 11:574485, (2020).</a></em></p>

<p><b>RNA-Puzzles Round IV: 3D structure predictions of four ribozymes and two aptamers.</b> <br />
  <b>Miao, Z.</b>, Adamiak, R. W., Antczak, M., Boniecki, M. J., Bujnicki, J., Chen, S.-J., Cheng, C. Y., Cheng, Y., Chou, F.-C., Das, R., Dokholyan, N. V., Ding, F., Geniesse, C., Jiang, Y., Joshi, A., Krokhotin, A., Magnus, M., Mailhot, O., Major, F., Mann, T. H., Piątkowski, P., Pluta, R., Popenda, M., Sarzynska, J., Sun, L., Szachniuk, M., Tian, S., Wang, J., Wang, J., Watkins, A. M., Wiedemann, J., Xiao, Y., Xu, X., Yesselman, J. D., Zhang, D., Zhang, Y., Zhang, Z., Zhao, C., Zhao, P., Zhou, Y., Zok, T., Żyła, A., Ren, A., Batey, R. T., Golden, B. L., Huang, L., Lilley, D. M., Liu, Y., Patel, D. J. &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32371455/">RNA 26, 982–995 (2020).</a></em></p>

<p><b>Putative cell type discovery from single-cell gene expression data.</b> <br />
  <b>Miao, Z.</b>, Moreno, P., Huang, N., Papatheodorou, I., Brazma, A.# &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/32424270/">Nat. Methods 17, 621–628 (2020).</a></em></p>

<p><b>RNA-Puzzles toolkit: a computational resource of RNA 3D structure benchmark datasets, structure manipulation, and evaluation tools.</b> <br />
  Magnus, M., Antczak, M., Zok, T., Wiedemann, J., Lukasiak, P., Cao, Y., Bujnicki, J. M., Westhof, E., Szachniuk, M. &amp; <b>Miao, Z.#</b> <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/31799609/">Nucleic Acids Res. 48, 576–588 (2020).</a></em></p>

<p><b>Decoding human fetal liver haematopoiesis.</b> <br />
  Popescu, D.-M., Botting, R. A., Stephenson, E., Green, K., Webb, S., Jardine, L., Calderbank, E. F., Polanski, K., Goh, I., Efremova, M., Acres, M., Maunder, D., Vegh, P., Gitton, Y., Park, J.-E., Vento-Tormo, R., <b>Miao, Z.</b>, Dixon, D., Rowell, R., McDonald, D., Fletcher, J., Poyner, E., Reynolds, G., Mather, M., Moldovan, C., Mamanova, L., Greig, F., Young, M. D., Meyer, K. B., Lisgo, S., Bacardit, J., Fuller, A., Millar, B., Innes, B., Lindsay, S., Stubbington, M. J. T., Kowalczyk, M. S., Li, B., Ashenberg, O., Tabaka, M., Dionne, D., Tickle, T. L., Slyper, M., Rozenblatt-Rosen, O., Filby, A., Carey, P., Villani, A.-C., Roy, A., Regev, A., Chédotal, A., Roberts, I., Göttgens, B., Behjati, S., Laurenti, E., Teichmann, S. A.# &amp; Haniffa, M.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/31597962/">Nature 574, 365–371 (2019).</a></em></p>

<p><b>BBKNN: Fast Batch Alignment of Single Cell Transcriptomes.</b> <br />
  Polański, K.^, Park, J.-E.^, Young, M. D.^, <b>Miao, Z.</b>, Meyer, K. B. &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/31400197/">Bioinformatics 36(3):964-965 (2019).</a></em></p>

<p><b>AbRSA: A robust tool for antibody numbering.</b> <br />
  Li, L.^, Chen, S.^, <b>Miao, Z.^</b>, Liu, Y., Liu, X., Xiao, Z.-X.# &amp; Cao, Y.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/31020723/">Protein Sci. 28, 1524–1531 (2019).</a></em></p>

<p><b>Exome sequencing identifies a novel missense variant in CTSC causing nonsyndromic aggressive periodontitis.</b> <br />
  Molitor, A., Prud’homme, T., <b>Miao, Z.</b>, Conrad, S., Bloch-Zupan, A., Pichot, A., Hanauer, A., Isidor, B., Bahram, S. &amp; Carapito, R.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/31068678/">Journal of Human Genetics 64, 689–694 (2019).</a></em></p>

<p><b>Comparative analysis of sequencing technologies for single-cell transcriptomics.</b> <br />
  Natarajan, K. N.#^, <b>Miao, Z.^</b>, Jiang, M., Huang, X., Zhou, H., Xie, J., Wang, C., Qin, S., Zhao, Z., Wu, L., Yang, N., Li, B., Hou, Y., Liu, S. &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/30961669/">Genome Biol. 20(1):70 (2019).</a></em></p>

<p><b>A test metric for assessing single-cell RNA-seq batch correction.</b> <br />
  Büttner, M.^, <b>Miao, Z.^</b>, Wolf, F. A., Teichmann, S. A.# &amp; Theis, F. J.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/30573817/">Nat. Methods 16, 43–49 (2019).</a></em></p>

<p><b>Genome-wide analyses reveal the IRE1a-XBP1 pathway promotes T helper cell differentiation by resolving secretory stress and accelerating proliferation.</b> <br />
  Pramanik, J., Chen, X., Kar, G., Henriksson, J., Gomes, T., Park, J.-E., Natarajan, K., Meyer, K. B., <b>Miao, Z.</b>, McKenzie, A. N. J., Mahata, B.# &amp; Teichmann, S. A.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/30355343/">Genome Med. 10, 76 (2018).</a></em></p>

<p><b>Evaluation of Protein-Ligand Docking by Cyscore.</b> <br />
  Cao, Y., Dai, W. &amp; <b>Miao, Z.</b> <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/29594775/">Methods in Molecular Biology 1762:233–243 (2018).</a></em></p>

<p><b>Mutations in signal recognition particle SRP54 cause syndromic neutropenia with Shwachman-Diamond-like features.</b> <br />
  Carapito, R., Konantz, M., Paillard, C., <b>Miao, Z.</b>, Pichot, A., Leduc, M. S., Yang, Y., Bergstrom, K. L., Mahoney, D. H., Shardy, D. L., Alsaleh, G., Naegely, L., Kolmer, A., Paul, N., Hanauer, A., Rolli, V., Müller, J. S., Alghisi, E., Sauteur, L., Macquin, C., Morlon, A., Sancho, C. S., Amati-Bonneau, P., Procaccio, V., Mosca-Boidron, A.-L., Marle, N., Osmani, N., Lefebvre, O., Goetz, J. G., Unal, S., Akarsu, N. A., Radosavljevic, M., Chenard, M.-P., Rialland, F., Grain, A., Béné, M.-C., Eveillard, M., Vincent, M., Guy, J., Faivre, L., Thauvin-Robinet, C., Thevenon, J., Myers, K., Fleming, M. D., Shimamura, A., Bottollier-Lemallaz, E., Westhof, E., Lengerke, C., Isidor, B. &amp; Bahram, S.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/28972538/">J. Clin. Invest. 127, 4090–4103 (2017).</a></em></p>

<p><b>RNA Structure: Advances and Assessment of 3D Structure Prediction.</b> <br />
  <b>Miao, Z.#</b> &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/28375730/">Annu. Rev. Biophys. 46, 483–503 (2017).</a></em></p>

<p><b>RNA-Puzzles Round III: 3D RNA structure prediction of five riboswitches and one ribozyme.</b> <br />
  <b>Miao, Z.</b>, Adamiak, R. W., Antczak, M., Batey, R. T., Becka, A. J., Biesiada, M., Boniecki, M. J., Bujnicki, J. M., Chen, S.-J., Cheng, C. Y., Chou, F.-C., Ferré-D’Amaré, A. R., Das, R., Dawson, W. K., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Geniesse, C., Kappel, K., Kladwang, W., Krokhotin, A., Łach, G. E., Major, F., Mann, T. H., Magnus, M., Pachulska-Wieczorek, K., Patel, D. J., Piccirilli, J. A., Popenda, M., Purzycka, K. J., Ren, A., Rice, G. M., Santalucia, J., Jr, Sarzynska, J., Szachniuk, M., Tandon, A., Trausch, J. J., Tian, S., Wang, J., Weeks, K. M., Williams, B., 2nd, Xiao, Y., Xu, X., Zhang, D., Zok, T. &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/28138060/">RNA 23, 655–672 (2017).</a></em></p>

<p><b>Quantifying side-chain conformational variations in protein structure.</b> <br />
  <b>Miao, Z.#</b> &amp; Cao, Y.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/27845406/">Sci. Rep. 6, 37024 (2016).</a></em></p>

<p><b>RBscore&amp;NBench: a high-level web server for nucleic acid binding residues prediction with a large-scale benchmarking database.</b> <br />
  <b>Miao, Z.#</b> &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/27084939/">Nucleic Acids Res. 44, W562–7 (2016).</a></em></p>

<p><b>DRSP: a Structural Database for Single Residue Substitutions in PDB.</b> <br />
  Liu, J.^, <b>Miao, Z.^</b>, Li, L., Xiao, Z. &amp; Cao, Y.# <br /><em><a href="https://www.researchgate.net/publication/311266971_DRSP_A_structural_database_for_single_residue_Substitutions_in_PDB">pibb.2016.0056</a></em></p>

<p><b>Prediction of nucleic acid binding probability in proteins: a neighboring residue network based score.</b> <br />
  <b>Miao, Z.</b> &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/25940624/">Nucleic Acids Res. 43, 5340–5351 (2015).</a></em></p>

<p><b>RNA-Puzzles Round II: assessment of RNA structure prediction programs applied to three large RNA structures.</b> <br />
  <b>Miao, Z.</b>, Adamiak, R. W., Blanchet, M.-F., Boniecki, M., Bujnicki, J. M., Chen, S.-J., Cheng, C., Chojnowski, G., Chou, F.-C., Cordero, P., Cruz, J. A., Ferré-D’Amaré, A. R., Das, R., Ding, F., Dokholyan, N. V., Dunin-Horkawicz, S., Kladwang, W., Krokhotin, A., Lach, G., Magnus, M., Major, F., Mann, T. H., Masquida, B., Matelska, D., Meyer, M., Peselis, A., Popenda, M., Purzycka, K. J., Serganov, A., Stasiewicz, J., Szachniuk, M., Tandon, A., Tian, S., Wang, J., Xiao, Y., Xu, X., Zhang, J., Zhao, P., Zok, T. &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/25883046/">RNA 21, 1066–1084 (2015).</a></em></p>

<p><b>A Large-Scale Assessment of Nucleic Acids Binding Site Prediction Programs.</b> <br />
  <b>Miao, Z.</b> &amp; Westhof, E.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/26681179/">PLoS Comput. Biol. 11, e1004639 (2015).</a></em></p>

<p><b>Modeling of protein side-chain conformations with RASP.</b> <br />
  <b>Miao, Z.</b>, Cao, Y. &amp; Jiang, T.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/24573473/">Methods Mol. Biol. 1137, 43–53 (2014).</a></em></p>

<p><b>RNA structure analysis of human spliceosomes reveals a compact 3D arrangement of snRNAs at the catalytic core.</b> <br />
  Anokhina, M., Bessonov, S., <b>Miao, Z.</b>, Westhof, E., Hartmuth, K. &amp; Lührmann, R.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/24002212/">EMBO J. 32, 2804–2818 (2013).</a></em></p>

<p><b>RNA-Puzzles: a CASP-like evaluation of RNA three-dimensional structure prediction.</b> <br />
  Cruz JA, Blanchet MF, Boniecki M, Bujnicki JM, Chen SJ, Cao S, Das R, Ding F, Dokholyan NV, Flores SC, Huang L, Lavender CA, Lisi V, Major F, Mikolajczak K, Patel DJ, Philips A, Puton T, Santalucia J, Sijenyi F, Hermann T, Rother K, Rother M, Serganov A, Skorupski M, Soltysinski T, Sripakdeevong P, Tuszynska I, Weeks KM, Waldsich C, Wildauer M, Leontis NB, Westhof E.# <br /><em><a href="https://rnajournal.cshlp.org/content/18/4/610.long/">RNA. 2012 Apr;18(4):610-25.</a></em></p>

<p><b>RASP: rapid modeling of protein side chain conformations.</b> <br />
  <b>Miao, Z.</b>, Cao, Y. &amp; Jiang, T.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/21949272/">Bioinformatics 27, 3117–3122 (2011).</a></em></p>

<p><b>Improved side-chain modeling by coupling clash-detection guided iterative search with rotamer relaxation.</b> <br />
  Cao, Y., Song, L., <b>Miao, Z.</b>, Hu, Y., Tian, L. &amp; Jiang, T.# <br /><em><a href="https://pubmed.ncbi.nlm.nih.gov/21216772/">Bioinformatics 27, 785–790 (2011).</a></em></p>


</div>

      </div>
      <!-- 侧边栏区域：占用 4 栏 -->
      <div id="newsid" class="col-md-4">
        <!-- Begin: Sidebar 模块 -->
<aside class="sidebar col-sm-44" style="padding: 15px;">
	    <!-- CSS Files -->
		<link href="/css/bootstrap.min.css" rel="stylesheet" type="text/css">
		<link href="/css/stylish-portfolio.css" rel="stylesheet" type="text/css">
		<link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
		<link href="//jpswalsh.github.io/academicons/css/academicons.css" rel="stylesheet">
		<link href="/stylesheets/screen.css" media="screen, projection, print" rel="stylesheet" type="text/css">
	<!-- Begin: 关于信息部分 -->
	<section>
		<h1>RNA-Puzzles</h1>
		<p>A collective blind evaluation of RNA three-dimensional structure prediction</p>
		<!-- GitHub 链接 -->
		<a href="https://github.com/RNA-Puzzles">
			<i class="fa fa-github-square fa-fw fa-2x"></i>
		</a>
		<!-- Email 链接 -->
		<a href="mailto:ibmc.cnrs@gmail.com">
			<i class="fa fa-envelope-square fa-fw fa-2x"></i>
		</a>
		<!-- Twitter 链接 -->
		<a href="https://twitter.com/RNAPuzzles">
			<i class="fa fa-twitter-square fa-fw fa-2x"></i>
		</a>
		<!-- Coffee 链接（示例链接，可根据需要修改） -->
		<a href="">
			<i class="fa fa-coffee fa-fw fa-2x"></i>
		</a>



	</section>
	<!-- End: 关于信息部分 -->

	<!-- Begin: 最近文章模块 -->
	<section class="panel panel-info">
		<div class="panel-heading">
			<h3 class="panel-title">Recent Posts</h3>
		</div>
		<div id="recent_posts" class="list-group">
			<a class="list-group-item" href="/news/2023/10/07/Submission-Process/">Submission Process</a>
			<a class="list-group-item" href="/news/2023/10/07/Research-Spectrum/">Research Spectrum</a>
			<a class="list-group-item" href="/news/2023/10/07/Recent-Engagements/">Recent Engagements</a>
			<a class="list-group-item" href="/news/2023/10/07/Participating-Teams/">Participating Teams</a>
			<a class="list-group-item" href="/news/2023/10/07/Methodological-Progress/">Methodological Progress</a>
		</div>
	</section>
	<!-- End: 最近文章模块 -->

	<!-- Begin: GitHub Repos 模块 -->
	<section class="panel panel-info clearfix">
		<div class="panel-heading">
			<h3 class="panel-title">GitHub Repos</h3>
		</div>
		<div class="list-group" id="gh_repos">
			<p class="loading">Status updating...</p>
		</div>
		<div class="gh-profile-link pull-right text-muted">
			<a href="https://github.com/RNA-Puzzles">@RNA-Puzzles</a> on GitHub
		</div>
		<!-- 使用 jQuery 动态加载 GitHub 仓库信息 -->
		<script type="text/javascript">
			$(document).ready(function(){
				// 如果未加载 jXHR 库，则动态加载
				if (!window.jXHR){
					var jxhr = document.createElement('script');
					jxhr.type = 'text/javascript';
					jxhr.src = '/javascripts/libs/jXHR.js';
					var s = document.getElementsByTagName('script')[0];
					s.parentNode.insertBefore(jxhr, s);
				}
				// 调用 github.showRepos() 函数显示仓库列表
				github.showRepos({
					user: 'RNA-Puzzles',
					count: 0,
					skip_forks: true,
					target: '#gh_repos'
				});
			});
		</script>
		<script src="/javascripts/github.js" type="text/javascript"></script>
	</section>
	<!-- End: GitHub Repos 模块 -->

	<!-- Begin: 分类模块 -->
	<section class="panel panel-info clearfix">
		<div class="panel-heading">
			<h3 class="panel-title">Categories</h3>
		</div>
		<div class="list-group-item">
			<a href="/blog/group_category/">group (16)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/news_category/">news (7)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/results_category/">results (39)</a>
		</div>
		<div class="list-group-item">
			<a href="/blog/tables_category/">table (129)</a>
		</div>
	</section>
	<!-- End: 分类模块 -->
</aside>
<!-- End: Sidebar 模块 -->

      </div>
    </div>
  </div>

    <div id="footer" class="panel">
  <div class="panel-footer" style='background-color: #333333; color: #eeeeee;'>
	<div class="container-fluid">
	  <div class="row">
		<div class="col-sm-4">
			
		  <p>&copy 2025 RNA-Puzzles. </p>
		  <p>We are part of the <a href="https://gzlab.ac.cn">Guangzhou Laboratory</a> and <a href="http://www.rnapuzzles.org/">RNApuzzles</a>.</p>
		  <p>Site made with <a href="https://jekyllrb.com">Jekyll</a>;  <a href="/aboutwebsite.html">copy and  modify it for your own research group.</a></p>
		  <p>  </p>
		  <p>  </p>
		</div>
		
		<div class="col-sm-4">
		  Funding:<br />
          - <a href="https://gzlab.ac.cn">R&D Programs of Guangzhou Laboratory</a><br />
          - <a href="http://www.most.gov.cn/">MOST IT+BT projects</a> <br />
          - <a href="https://www.nsfc.gov.cn/">NSFC</a> <br />	  
		</div>
		
		<div class="col-sm-4">
		  Contact:<br />
		  Building B2, Guangzhou National Laboratory<br />
		  96 Xingdao South Road, Guangzhou International Bio Island, <br />
		  Haizhu District, Guangzhou, Guangdong, China.<br />
          (<a href="https://goo.gl/maps/hDLr6siEAS2Eaypt8">Maps</a>)
		</div>
	  </div>
	</div>
  </div>
</div>

<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
<script src="/js/bootstrap.min.js"></script>


  </body>

</html>
